README.md

```diff
@@ -32,14 +32,15 @@
 * ansible.cfg: User customized Ansible configuration file
 * autoinstall: Folder for guest OS unattend install configuration files
 * common: Folder for common tasks called in test cases
-* docs: Folder for guide files
+* docs: Folder for guide file and known issues
 * env_setup: Folder for playbooks or tasks which to prepare or cleanup testing environment
 * linux: Folder for playbooks to test Linux guest OS
 * Windows: Folder for playbooks to test Windows guest OS
```

Comment:
The first letter of windows folder should be lower case

Reply:
Right, modified.